prometheus/templates/prometheus.yml.tmpl

```diff
@@ -31,6 +31,14 @@ scrape_configs:
       port: ${app.port}
 %{ endfor ~}
 
+%{ if billing_exporter_endpoint != "" }
```

Comment:
why not include it as part of the existing var.exporters? and have a default scarpe_internval for others?

Reply:
done